packages/lib/modules/lbp/steps/useCreateLbpStep.tsx

```diff
@@ -77,36 +88,36 @@ export function useCreateLbpStep(): TransactionStep {
   const blockProjectTokenSwapsIn = userActions === 'buy_only' ? true : false
 
   const { symbol: launchTokenSymbol } = useTokenMetadata(launchTokenAddress, selectedChain)
-  const { symbol: collateralTokenSymbol } = useTokenMetadata(collateralTokenAddress, selectedChain)
+  const { symbol: reserveTokenSymbol } = useTokenMetadata(reserveTokenAddress, selectedChain)
```

Comment:
i suggest we do the variable name switch everywhere so it's in sync with the contracts (can use whatever (marketing) name is best for the ui though)

Reply:
i strongly agree but @agorer initially pushed back against matching onchain. can't remember the reason tho

Reply:
i would _not_ use the business vocabulary as that is more subject to change than the backend/contracts
also interactions with backend/contracts (on everything) is more frequent than with users (so i think it will be less translating that way instead of more)